packages/llmRunner/src/llmRunner/Artifact.ts

```diff
@@ -0,0 +1,30 @@
+import { CodeState } from "./LLMStep";
+
+export type Artifact =
```

Comment:
Artifacts are plain objects right now but I imagine they'll evolve into class instances eventually.

Also, artifacts need unique ids — then, I hope, we'll be able to wire steps into a execution graph and render it through react-flow.